### DISCUSSION TITLE
[FR] Jumper R900 telemetry.

### DISCUSSION THREAD

**Discussion by Tygrys-1** [iNavFlight/inav#6367] [Q&A] [answered]
The Jumper R900 receiver is designed to handle Pixhawk telemetry. I belive it is the SPort protocol, but divided into RX/TX, to avoid the need of the the external UART to SPort converter for Pixhawk. Can the INav feed the SPort telemetry to such solution? 
The same applies to Jumper R8 receiver.
https://www.jumper-b2b.com/jumper-r900-receiver-900mhz-long-range-rx-p0082.html

**Comment by DzikuVx** [comment]
I  belive that it can work if you select half duplex OFF. That should put the port to separate RX/TX line. But will that work with this receiver, I do not know

**Reply by Tygrys-1** [reply]
Thank you for the reply. Can it be done by some CLI magic? I do not have the receiver, as there is no sense to buy them if they can not work with the telemetry (ok, there can be the hardware SPort mod, but it is less sense to encode the SPORT to one line in the iNav software and then decode it by hardware to fit into the receiver).

**Comment by DzikuVx** [comment]
Just use latest INAV and use Receiver Tab in configurator to set it

**Reply by Tygrys-1** [reply]
The receiver protocol is SBUS on one port. The telemetry is second port with SPort. Which port will affect the half duplex setting ?

**Comment by Tygrys-1** [comment, marked as answer]
Ok, found the solution on the telemetry.md. I need to set the telemetry to external hardware inverter, and this is possible at current release of iNav. Thanks Paweł for guiding me to this solution.

**Reply by ibonFPV** [reply]
Hi i have bought an R900 and I dont know how to conect  the telemetry , i have conected de Sbus to the flight controller and it works but for telemetry what i have to coenct ?, "pix tx" or "pix rx" or both, thanks you very much for your help

**Reply by Tygrys-1** [reply]
Both lines. Then write in CLI
`set telemetry_uart_unidir = ON
`
`set telemetry_inverted = ON`

**Reply by ibonFPV** [reply]
Thanks for the help, but the physical connections like, what receiver cables are needed and where would they have to be connected? thank you

**Reply by Tygrys-1** [reply]
PixTX to serial TX and PixRX to serial RX of the same serial port. Set the smartport telemetry on that port in ports tab.

**Reply by geofrancis** [reply]
did you ever get this to work, I have tried everything and got nowhere, the option set telemetry_uart_unidir = ON isnt even in new versions of inav, i had to go back to v2.3 but it still wont work.
